### PR TITLE
Skip storing draft in RBF form, reintroduce precomposedForm

### LINKED ISSUE #13325: Cancelled bump fee modal creates send draft
**Describe the bug**
When I cancel bump fee tx review modal and I navigate to send form, it is partially filled and symbol is missing. I would not expect filled form at all!

If the tx is mined with old fee and user will want to send another tx in send form, it will not work as it adds also rbf params

**Info:**

-   Suite Version: 24.7.1

**How to reproduce**
1. Send tx with low gas price
1. Bump fee
2. Cancel it in review tx modal
3. Navigate to send form

**Screenshots**
Screenshot 2024-07-14 at 20 55 33

### LINKED ISSUE #14753: Spending right after the bump of RBF results in "Transaction signing error"
Steps:

1. Create transaction with 1sat/byte (to never be confirmed)
2. Bump it to 2sats/byte
3. Try to spend the UTXO of the 2. tx (use coin control
4. -> error
```
Transaction signing error: Transaction 'd06dfa184166eed133240ed4f712f43db13f500fc19b7d1597e10e09dd849f1b' not found
```

image

The clearing of the send-form solves the issue. 
image

### LINKED ISSUE #15114: Transaction signing error
**Describe the bug**
Transaction signing error: transformOrigTransactions: invalid input at 49b7518c5a8e71ab3a3eaeaca4ea56d0f2478ae0f2

**Info:**

-   Remembered Wallet: yes
-   Suite Version: web
-   Browser: Microsoft edge
-   Browser Installation Method: , windows store
-   OS: Microsoft
-   Bridge: wifi
-   Firmware Version: KBG40ZNS256G_BG4A_KIOXIAAEMS0104

**How to reproduce**

1. Click on '....'
1. Scroll down to '....'
1. See error

**Expected behavior**
I was trying to send my coins to a platform (Binance or Bybit) but the attached message keep poping-up. 

**Screenshots**
Screenshot 2024-10-30 125839

**Additional context**

### PR DESCRIPTION
**Pull request by komret**
RBF form should not store transaction drafts. It creates confusion when send form is opened with unexpected pre-filled data and also causes errors when a transaction is created in thusly affected send form.

This PR reintroduces `precomposedForm` which holds similar data as drafts, but is used by both send form and RBF form synchronize data between the form and the review modal.

## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/13325
Resolve https://github.com/trezor/trezor-suite/issues/14753 (create a transaction, open RBF form, **replace the transaction**, go to send form, try to send)
Resolve https://github.com/trezor/trezor-suite/issues/15114 (create a transaction, open RBF form, **close it without replacing the transaction**, go to send form, try to send)

### CODE REVIEW

**Comment by PeKne** [comment]
This PR do no harm to the mobile implementation. Only what changes for us is that there will be an extra attribute in the redux state, but we do not interact with it in any way. So feel free to merge this change.  I tested your branch locally on mobile and it works as expected 👌.

File: packages/suite/src/actions/wallet/send/sendFormThunks.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ import { findLabelsToBeMovedOrDeleted, moveLabelsForRbfAction } from '../moveLab
 export const MODULE_PREFIX = '@send';
 
 export const saveSendFormDraftThunk = createThunk(
-    `${MODULE_PREFIX}/saveSendFormDraftThunk`,
+    `${MODULE_PREFIX}/ `,
```
Comment (PeKne) [inline packages/suite/src/actions/wallet/send/sendFormThunks.ts]: This is probably not intentional

Comment (komret) [reply packages/suite/src/actions/wallet/send/sendFormThunks.ts]: 27dab3bed715b87cd0b1ec77eb818a99c9355c23

**Comment by komret** [comment]
@PeKne Thanks! Consider using the `precomposedForm` though, you might need it when you implement the "Bump fee" functionality.

**Review by peter-sanderson — APPROVED** [review]

**Comment by bosomt** [comment]
**QA OK**

✅ send TEST tx
✅ abort bump tx - send modal is empty
✅ bump tx - send modal empty 
✅ sent, bumped , from same account i sent "change" from previous unconfirmed transaction
✅ bumped first tx again , no problem, chained tx was removed
✅ prepare send transaction, leave, come back, prepared tx is still present 
✅ clear all button works as expcted


**Info:**
 - Suite version: desktop 24.12.0 (377d496a30264c9048f6f3ac0f5e90575cb4b11a)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuiteDev/24.12.0 Chrome/130.0.6723.91 Electron/33.1.0 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T3T1 2.8.3 regular (revision 7f373ae71eca855dd41b4a0fdcc7cadaa13a8281)
 - Transport: BridgeTransport 3.0.0-bundled.24.12.0

**Pull request merged**